src/ocean/util/app/ext/ReopenableFilesExt.d

```diff
@@ -114,6 +178,36 @@ public class ReopenableFilesExt : IApplicationExtension, ISignalExtExtension
 
     /***************************************************************************
 
+        Reopens the specified log file.
+
+        Params:
+            file_path = path of the file to open
+
+        Returns:
+            true if the file was registered and reopen, false otherwise.
+
+    ***************************************************************************/
+
+    public bool reopenFile ( cstring file_path )
+    {
+        foreach (file; this.open_files)
+        {
+            path_buffer.length = 0;
```

Comment:
I'd recommend `path_buffer.reset()`, it makes it more obvious you don't need `enableStomping` :)

Reply:
Yeah I started typing `enableStomping` and had the "oh..." moment :P

Reply:
The sole purpose of Buffer :-P